packages/react-instantsearch-theme-algolia/styles/_Panel.scss

```diff
@@ -0,0 +1,10 @@
+.ais-Panel__empty {
+  display: none;
```

Comment:
I just want to ensure that we tested the "do not hide everything" at least once on our examples and can decide if it's better or not than the hide everything strategy.